bsi/src/test/java/org/roaringbitmap/bsi/R64NavigableBSITest.java

```diff
@@ -0,0 +1,475 @@
+package org.roaringbitmap.bsi;
+
+import org.roaringbitmap.bsi.longlong.Roaring64NavigableMapSliceIndex;
+import org.roaringbitmap.longlong.Roaring64NavigableMap;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.LongStream;
+
+public class R64NavigableBSITest {
```

Comment:
Should we share tests between `R64BSITest` and `R64NavigableBSITest `, rather than duplicating them?

Reply:
maybe ok, but i'm not quite sure about this is a test for a new BSI impl (although most of the test cases are similar).

Reply:
I don't think there is a strong need to merge the two tests. what do you think? @blacelle 
if we merge, we could not reduce duplicate code